libs/hdf-converters/src/snyk-mapper.ts

```diff
@@ -37,142 +39,467 @@ function nistTag(identifiers: unknown[]): string[] {
+                  transformer: nistTag
+                },
+                cweid: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? 'issueData.' : ''
+                  }identifiers.CWE`,
+                  transformer: (input: unknown) => input || []
+                },
+                cveid: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? 'issueData.' : ''
+                  }identifiers.CVE`,
+                  transformer: (input: unknown) => input || []
+                },
+                ghsaid: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? 'issueData.' : ''
+                  }identifiers.GHSA`,
+                  transformer: (input: unknown) => input || []
+                },
+                exploitMaturity: {
+                  path: `${isApi?.apiVersion === 1 ? 'issueData.' : ''}exploit${
+                    isApi?.apiVersion === 1 ? 'Maturity' : ''
+                  }`
+                },
+                CVSSv3: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? 'issueData.' : ''
+                  }identifiers.CVSSv3`
+                },
+                cvssScore: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? 'issueData.' : ''
+                  }identifiers.cvssScore`
+                }
+              },
+              refs: [],
+              source_location: {},
+              title: {
+                path: `${isApi?.apiVersion === 1 ? 'issueData.' : ''}title`
+              },
+              id: {path: 'id'},
+              desc: {
+                path: `${
+                  isApi?.apiVersion === 1 ? 'issueData.' : ''
+                }description`
+              },
+              impact: {
+                path: `${isApi?.apiVersion === 1 ? 'issueData.' : ''}severity`,
+                transformer: impactMapping(IMPACT_MAPPING)
+              },
+              code: {
+                transformer: (item: Record<string, unknown>): string => {
+                  return JSON.stringify(item, null, 2);
+                }
+              },
+              results: [
+                {
+                  status: ExecJSON.ControlResultStatus.Failed,
+                  code_desc: {
+                    ...(!isApi && {path: 'from'}),
+                    transformer: (input: unknown): string => {
+                      let paths = input;
+                      if (isApi?.apiVersion === 1) {
+                        paths = Object.values(
+                          _.pick(
+                            _.get(this.unconvertedData, 'dependencyChains'),
+                            (_.get(input, 'pkgVersions') as string[]).map(
+                              (version) =>
+                                `${_.get(input, 'pkgName')}@${version}`
+                            )
+                          )
+                        );
+                      }
```

Comment:
Results `code_desc` feels lacklustre.

Reply:
I dunno what you mean by this lmao.  please give actionable advice in the future.